src/options.ts

```diff
@@ -1012,6 +1012,9 @@ export interface IsURLOptions {
   validate_length?: boolean;
   allow_fragments?: boolean;
   allow_query_components?: boolean;
+  allow_numeric_tld?: boolean;
+  allow_wildcard?: boolean;
+  ignore_max_length?: boolean;
```

Comment:
Hey, WDYT of making `IsURLOptions` extend from `IsFQDNOptions`?
We can then remove a couple of the repeated properties from here.

Reply:
Hey, OK!